lib/liquid/usage.rb

```diff
@@ -0,0 +1,6 @@
+module Liquid
+  module Usage
+    def self.increment(name, sample_rate: 0.1, tags: {})
```

Comment:
`tags: {}` will allocate a new hash on each call, even though the argument isn't used in this stub implementation.  You could move that default value into a constant to avoid that problem:

```suggestion
    EMPTY_HASH = {}.freeze
    def self.increment(name, sample_rate: 0.1, tags: EMPTY_HASH)
```

Do we intend to use the tags in the future?  Note that tags are a datadog extension to the statsd protocol.